recipes/biobear/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "biobear" %}
+{% set version = "0.7.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biobear-{{ version }}.tar.gz
+  sha256: 6fb47bee0ff81d99a687ffd316e1be5549a4653c1b8ff1c6b7e4d500bea5b228
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
```

Comment:
```suggestion
    - {{ compiler('rust') }}  # [build_platform != target_platform]
```

Reply:
I committed this suggestion, but it seems to result in an error that cargo is no longer found...

```
Processing $SRC_DIR
  Added file://$SRC_DIR to build tracker '/private/tmp/pip-build-tracker-l58jdo74'
  Created temporary directory: /private/tmp/pip-modern-metadata-rrfepg6q
  Preparing metadata (pyproject.toml): started
  Running command Preparing metadata (pyproject.toml)
  Checking for Rust toolchain....

  Cargo, the Rust package manager, is not installed or is not on PATH.
  This package requires Rust and Cargo to compile extensions. Install it through
  the system's package manager or via https://rustup.rs/

  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /Users/runner/mambaforge/conda-bld/biobear_1689783999438/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/python /Users/runner/mambaforge/conda-bld/biobear_1689783999438/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpwkixtdki
  cwd: /Users/runner/mambaforge/conda-bld/biobear_1689783999438/work
  Preparing metadata (pyproject.toml): finished with status 'error'
error: metadata-generation-failed

```